js/components/ConsortReport.vue

```diff
@@ -57,9 +57,7 @@ export default {
      */
```

Comment:
Since this doesn't fetch config anymore, it should be renamed. You should also remove `data.reportConfig` if it isn't being used.